src/server/pps/pretty/pretty.go

```diff
@@ -70,6 +72,58 @@ func PrintJobInfo(w io.Writer, jobInfo *ppsclient.JobInfo, fullTimestamps bool)
 	fmt.Fprintln(w)
 }
 
+// PrintJobSetInfo pretty-prints jobset info.
+func PrintJobSetInfo(w io.Writer, jobSetInfo *ppsclient.JobSetInfo, fullTimestamps bool) {
+	// Aggregate some data to print from the jobs in the jobset
+	success := 0
+	failure := 0
+	var created *types.Timestamp
+	var modified *types.Timestamp
+	for _, job := range jobSetInfo.Jobs {
+		switch job.State {
+		case ppsclient.JobState_JOB_SUCCESS:
+			success++
+		case ppsclient.JobState_JOB_FAILURE:
+			failure++
+		}
+
+		if created == nil {
+			created = job.Created
+			modified = job.Created
+		} else {
+			if job.Created.Compare(created) < 0 {
+				created = job.Created
+			}
+			if job.Created.Compare(modified) < 0 {
+				modified = job.Created
+			}
```

Comment:
I'm not sure what modified is meant to represent here. Do you want it to be the latest created time, I guess for deferred processing?